client/src/onePage/AboutUs/AboutUsTexts/AboutUsTexts.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const AboutUsTexts = () => {
+  return (
+    <div
+      className="container"
+      style={{
+        position: 'absolute',
+        justifyContent: 'center',
+        textAlign: 'center',
+        marginTop: '450px',
+        marginLeft: '80px',
+        color: 'white',
+      }}
+    >
+      <div className="text position-left">
+        <p>Surprising</p>
+        <p>and playful ways</p>
+        <p>to tell a story!</p>
+      </div>
+    </div>
```

Comment:
Maybe MUI components will be better to use, to be more consistent,

Reply:
Yeah, I have some idea with this by using a GSAP. More about this later!